UI/window-basic-main-profiles.cpp

```diff
@@ -862,6 +862,12 @@ void OBSBasic::CheckForSimpleModeX264Fallback()
 				name = SIMPLE_ENCODER_X264;
 				return false;
 			}
+		} else if (strcmp(name, SIMPLE_ENCODER_NVENC_AV1) == 0) {
+			if (!nve_supported) {
+				changed = true;
+				name = SIMPLE_ENCODER_X264;
```

Comment:
This should fallback to rav1e, svt, or aom here, shouldn't it? Otherwise this makes no sense.

Reply:
You know, I think I'd rather make it just explicitly fail than fall back to a software AV1 encoder. Software AV1 encoders aren't really usable for most people due to their core requirements

Reply:
Are we sure about that? Realtime AOM modes seemed to work reasonably okay on my PCs from 2014 (4th generation Intel) and 2016 (6th generation Intel).

Reply:
If it did, it probably didn't compress very well.

Reply:
But does that matter when you're streaming to the other end? Realtime encoding trades off compression for quality and speed.

Reply:
I suppose I'm fine with falling back to AOM or something.

Reply:
That would be great, I know I would certainly appreciate it!

Reply:
I don't think falling back from HW encoder to SW encoder is a good idea. Just select the SW encoder if you want to use that. Error out with a helpful message if we can't use the HW encoder.

Reply:
Also confirming falling back from a hardware to a software encoder is a bad idea. We don't fall back to x264 for any of the other hardware h264 encoders, and I don't see a reason why we should start doing that for those or AV1.

Reply:
I'm fine with no fallback at all too. Though I'd love to eventually see an AV1 alternative to the x264 plugin.

Reply:
We already have AOM and SVT.

Reply:
This code doesn't have anything to do with silently falling back. This code is for when you start up obs and the encoder no longer exists with the current config anymore (i.e. switched parts, copied config from another computer).